cpp/data/zotero_cgi/delivered.html

```diff
@@ -24,11 +24,11 @@
     </head>
     <body>
         <h1>Zotero Harvesting</h1>
-        <h2>Delivered records for Zeder ID {zeder_id} ({zeder_instance})</h2>
+        <h2>Delivered records for &quot;{journal_name}&quot;</h2>
```

Comment:
Why are you not using literal quotes here?

Reply:
Because i always try to use encoded entities when writing HTML text, to avoid later copy/paste errors e.g. when copying text into an attribute.

Reply:
I don't think that is very readable.  How often do you really copy text into an attribute and when does the reviewer not catch that?

Reply:
https://stackoverflow.com/a/46000063/13923163